src/Uno.UI/UI/Xaml/Shapes/Shape.cs

```diff
@@ -91,7 +91,7 @@ private void InvalidateForBrushChanged()
 #endif
 			}
 
-#region Stroke Dependency Property
+		#region Stroke Dependency Property
```

Comment:
you might want to exclude the changes to this file

Reply:
I might ... but as it was only few lines I thought it was clear enough and it would improve code quality